nixos/modules/services/networking/xrdp.nix

```
@@ -4,7 +4,7 @@ with lib;
 
 let
   cfg = config.services.xrdp;
-  confDir = pkgs.runCommand "xrdp.conf" { preferLocalBuild = true; } ''
+  confDir = pkgs.runCommand "xrdp.conf" { preferLocalBuild = true; } (''
```

the parenthesis are unnecessary
```suggestion
  confDir = pkgs.runCommand "xrdp.conf" { preferLocalBuild = true; } ''
```

---

nevermind they are, I'm tired... no more reviews for me xD